assets/javascripts/src/components/pages/Contribution.jsx

```diff
@@ -123,7 +125,7 @@ class FormWithRecurring extends React.Component {
 
 class FormWithoutRecurring extends React.Component {
     render() {
-        return <div className={'contribute-controls contribute-fields ' + (this.props.amounts.length % 3 ? 'option-button__fours' : 'option-button__three')}>
+        return <div className="contribute-controls contribute-fields option-button__fours">
```

Comment:
Thanks, I've also got rid of the SASS for option-button__threes, as it's now unused.